queries.sql

```diff
@@ -19,4 +19,64 @@ SELECT * from animals WHERE name != 'Gabumon';
 
 SELECT *
 FROM animals
-WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+
+
+UPDATE animals SET species = 'unspecified';
+
+BEGIN TRANSACTION;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species = 'unspecified';
+COMMIT TRANSACTION;
```

Comment:
- [ ] You are supposed to run `SELECT species from animals;` in order to check if the changes went through

Reply:
Hi @descholar-ceo ,
Thanks for the feedback, i'll do that.